### PR TITLE
Reconfigure spending tracks on Kusama

### PR DESCRIPTION
**Pull request by adamsteeber**
These reconfigurations are what many believe is the first step in changing the paradigm of Treasury spending. Discussion can be found here:  https://forum.polkadot.network/t/kusama-treasury-follow-up-analysis-continued-budgeting-incompetence.

This will change the following parameters:

| Track | Spend Limit| Max Deciding | Min Approval | Preparation Period | Confirmation Period | Decision Period |
|-------|--------------|----------------|-----------------|---------------------|------------------------|------------------|
|Small Tipper|No Change|200 -> 6|No Change|1 mins -> 3 hrs|No Change|No Change|
|Big Tipper|No Change|100 -> 3|No Change|10 mins -> 6 hrs|No Change|No Change|
|Small Spender|333 KSM -> 100 KSM|50 -> 7|No Change|4 hrs -> 24 hrs|No Change|No Change|
|Medium Spender|3,333 KSM -> 1,250 KSM|50 -> 2|No Change|4 hrs -> 24 hrs|No Change|No Change|
|Big Spender|33,333 KSM -> 2,500 KSM|50 -> 1|50% -> 66%|4 hrs -> 24 hrs|48 hrs -> 24 hrs|No Change|
|Treasurer|No Change|No Change|50% -> 66%|2 hrs -> 48 hrs|3 hrs -> 48 hrs|14 days -> 28 days|

This pull request coincides with [referendum 235](https://kusama.polkassembly.io/referenda/235) on Kusama which implements these changes in v9430. Should that referendum pass, this can be merged into v1.0.0 to ensure these changes persist.

### CODE REVIEW

File: runtime/kusama/src/governance/tracks.rs [hunk 1]
```diff
@@ -22,14 +22,20 @@ const fn percent(x: i32) -> sp_arithmetic::FixedI64 {
 	sp_arithmetic::FixedI64::from_rational(x as u128, 100)
 }
 use pallet_referenda::Curve;
-const APP_ROOT: Curve = Curve::make_reciprocal(4, 28, percent(80), percent(50), percent(100));
-const SUP_ROOT: Curve = Curve::make_linear(28, 28, percent(0), percent(50));
-const APP_STAKING_ADMIN: Curve = Curve::make_linear(17, 28, percent(50), percent(100));
+const APP_ROOT: Curve =
+	Curve::make_reciprocal(4, 28, percent(80), percent(50), percent(100));
```
Comment (ggwpez) [inline runtime/kusama/src/governance/tracks.rs:26]: The format changes make this a bit difficult to review. I will try `bot fmt` maybe it has write access.

**Comment by ggwpez** [comment]
bot fmt

File: runtime/kusama/src/governance/tracks.rs (hunk 1 above)
Comment (ggwpez) [reply runtime/kusama/src/governance/tracks.rs:26]: The bot has no access. Could you please format the code with the same version as in the CI, `cargo +nightly-2023-05-16 fmt`?

Comment (adamsteeber) [reply runtime/kusama/src/governance/tracks.rs]: Hi, sorry for the formatting! I've reverted the formatting back to what it used to be to make comparison easier.

**Comment by Polkadot-Forum** [comment]
This pull request has been mentioned on **Polkadot Forum**. There might be relevant details there:

https://forum.polkadot.network/t/kusama-treasury-follow-up-analysis-continued-budgeting-incompetence/3143/20

**Review by ggwpez — APPROVED** [review]
My approval is not count anyway since its a runtime file, but it looks fine to me from a technical perspective.

**Comment by ggwpez** [comment]
It probably still makes sense to open an RFC here https://github.com/polkadot-fellows/RFCs to get an on-chain support signal.  
That should help to clear out doubts about the exact values and clear the way to having it uncontroversial merged.

**Comment by bkchr** [comment]
@adamsteeber as @ggwpez already said, this should be included in a RFC. As there is already this RFC open: https://github.com/polkadot-fellows/RFCs/pull/20 I would please ask you to add your feedback there and then it can probably directly be integrated in this RFC. 

I'm going to close this pr, just to clean up the list of prs.

**Pull request closed without merging**